arrow/src/main/java/org/apache/iceberg/arrow/vectorized/VectorizedArrowReader.java

```diff
@@ -119,6 +123,10 @@ private enum ReadType {
     DICTIONARY
   }
 
+  protected Types.NestedField icebergField() {
```

Comment:
Exposing it to all readers so that we can construct typed constant vectors later.